conf/Makefile.px4

```diff
@@ -0,0 +1,36 @@
+# Hey Emacs, this is a -*- makefile -*-
+#
+# Copyright (C) 2013 The Paparazzi Team
+#
+# This file is part of paparazzi.
+#
+# paparazzi is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2, or (at your option)
+# any later version.
+#
+# paparazzi is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with paparazzi; see the file COPYING.  If not, see
+# <http://www.gnu.org/licenses/>.
+#
+
+#
+# This Makefile uses the generic Makefile.arm-linux and adds upload rules for the PixHawk
+#
+include $(PAPARAZZI_SRC)/conf/Makefile.stm32-no_upload
+
+ifeq ($(FLASH_MODE),PX4_BOOTLOADER)
```

Comment:
IMHO it would make more sense to add this to `conf/Makefile.stm32-upload` and remove `Makefile.px4` and `Makefile.stm32-no_upload`
That way there is no duplication and you also open the option to use other flash modes for px4 again...